components/Buttons/Button/Button.tsx

```diff
@@ -36,20 +36,53 @@ interface GoogleAnalyticsEventPropType {
   transport?: 'beacon' | 'xhr' | 'image';
 }
 
-type ButtonProps = {
+interface ButtonProps
+  extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'disabled'>,
+    VariantProps<typeof buttonCva> {
   /**
    * Helps track in-page `event` interactions.
    */
   analyticsObject?: GoogleAnalyticsEventPropType;
-  /**
-   * Forces the component's width as wide as its parent container's width.
-   */
-  fullWidth?: boolean;
-  /**
-   * Sets the button color theme.
-   */
-  theme?: 'primary' | 'secondary';
-} & ButtonHTMLAttributes<HTMLButtonElement>;
+}
+
+export const buttonCva = cva(
+  'inline-block font-din-condensed rounded leading-none border-4 border-solid cursor-pointer font-bold text-center uppercase py-4 px-3 whitespace-nowrap transition-all duration-200 ease-linear min-w-[175px]',
+  {
+    variants: {
+      theme: {
+        primary: 'bg-primary border-primary outline-primary text-secondary',
+        secondary: 'bg-secondary border-secondary outline-secondary text-primary',
+      },
+      fullWidth: {
```

Comment:
later refactor, but i dont think we need this prop/variant anymore... people can easily just do `className="w-full"` now. it was more valuable when CSS Modules